front/app/containers/Admin/invitations/all/index.tsx

```diff
@@ -1,16 +1,11 @@
-// libraries
```

Comment:
Refactored this to functional component, extracted the table header into `TableHeader.tsx`, and replaced the `semantic-ui` table stuff obviously